pkg/models/resources/v1alpha3/customresourcedefinition/customresourcedefinitions_test.go

```diff
@@ -20,7 +20,7 @@ import (
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
-	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
`extensionV1` may be better than `v1`.

Reply:
Sure. But this is generated by `goimports`, if we change it to `extensionV1 `, the `Github Action` will fail.